### ISSUE TITLE
rnacommender Fatal error: Exit code 1 ()

### ISSUE THREAD

**Issue by scholtalbers** [bgruening/galaxytools#388]
I just tried to install and run rnacommender from the main toolshed. Installation went fine (without any dependencies?). 
But upon running the tool it exits with a "Fatal error: Exit code 1 ()". This is caused, by the somewhat odd init.sh script that looks for a directory in the **job working directory** and than downloads the required data - which on our server fails.

Shouldn't the init script just be run once after installing, instead of on every tool run?

**Comment by bgruening** [comment]
@scholtalbers have you enabled conda dependencies in Galaxy?
This tool comes with Conda dependencies so you need to have a Galaxy version >=16.01 with conda enabled.

**Comment by scholtalbers** [comment]
@bgruening no, this is still running 15.10.. so I might not be able to run the script without providing the dependencies myself I guess.

However, the error does not come from missing dependencies (I assume), just from the init script and a failed download.

**Comment by bgruening** [comment]
My guess is that the download files because of missing dependencies. This is what the script is doing. And the error occurs because the file that is expected does not exists :(

We have a Docker Image with this tool and a newer Galaxy version if this is of any interest for you.
ping @gianlucacorrado

**Comment by scholtalbers** [comment]
Error is below (from the hide.txt). The script downloads a tar of ~350mb..(or at least tries)
The thing is that is should probably not check for the availability of the data in the working directory, but rather in the tool or tool-data directory.

```
>cat hide.txt 
--2016-08-12 12:56:39--  http://www.googledrive.com/host/0B9v5_ppcfmgWNTJzVjlkc0pCMVU
Resolving www.googledrive.com... 216.58.208.33, 2a00:1450:4001:815::2001
Connecting to www.googledrive.com|216.58.208.33|:80... connected.
HTTP request sent, awaiting response... 301 Moved Permanently
Location: https://www.googledrive.com/host/0B9v5_ppcfmgWNTJzVjlkc0pCMVU [following]
--2016-08-12 12:56:40--  https://www.googledrive.com/host/0B9v5_ppcfmgWNTJzVjlkc0pCMVU
Connecting to www.googledrive.com|216.58.208.33|:443... connected.
ERROR: certificate common name `*.googleusercontent.com' doesn't match requested host name `www.googledrive.com'.
To connect to www.googledrive.com insecurely, use `--no-check-certificate'.
tar: 0B9v5_ppcfmgWNTJzVjlkc0pCMVU: Cannot open: No such file or directory
tar: Error is not recoverable: exiting now
rm: cannot remove `0B9v5_ppcfmgWNTJzVjlkc0pCMVU': No such file or directory
```

**Comment by bgruening** [comment]
It's the model that is used by the tool, currently we just download it as part of the tool, which is suboptimal, but this is how @gianlucacorrado has designed it currently. We have plans to offer a data select box, so you can download the model by your own and distribute it via a Library, but at the moment it's downloading the model temporary during every tool run.
I will try this today, but without dependencies it will not work either.
